File: azure/transporterSolution/transporterService/DataObjects/OrderRequestItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using transporterService.DataTranspObjects;

namespace transporterService.DataObjects
{
    public partial class OrderRequestItem : BaseItem
    {
        public string DateTime { get; set; }
        public virtual LatLnRequestItem From { get; set; }
        public virtual LatLnRequestItem To { get; set; }
        public virtual UserItem User { get; set; }
        public int MaxDistance { get; set; }

        //response
        public int Results { get; set; }
        public virtual ICollection<RouteItem> RouteItems { get; set; }

        //public OrderRequestItem()
        //{
        //    RouteItems = new List<RouteItem>();
        //}

    }
}

File: android/app/src/main/java/com/example/fugliara/ccs002/server/dataobjects/OrderRequestItem.java
package com.example.fugliara.ccs002.server.dataobjects;

import java.util.List;

public class OrderRequestItem {
    @com.google.gson.annotations.SerializedName("id")
    public String id;
    @com.google.gson.annotations.SerializedName("from")
    public LatLnItem from;
    @com.google.gson.annotations.SerializedName("to")
    public LatLnItem to;
    @com.google.gson.annotations.SerializedName("maxDistance")
    public int maxDistance;
    @com.google.gson.annotations.SerializedName("user")
    public UserItem user;
    @com.google.gson.annotations.SerializedName("results")
    public int results;
    @com.google.gson.annotations.SerializedName("routeItems")
    public List<RouteItem> routeItens;

    public OrderRequestItem() {
        id = "";
        from = null;
        to = null;
        maxDistance = 0;
        user = null;
        results = 0;
        routeItens = null;
    }
}
